src/doc/en/developer/packaging.rst

```diff
@@ -1366,3 +1207,220 @@ must meet the following requirements:
   :ref:`chapter-github`.
 
 
+.. _section-spkg-patching:
+
+Modifying third-party code
+==========================
+
+In the Sage distribution, we try to use unpatched original upstream tarballs
+of stable versions of third-party packages whenever possible.
+
+Sometimes, however, modifications are necessary, either to fix a bug or
+to make the package build on the platforms supported by Sage.
+
```

Comment:
Maybe the two previous paragraphs should be one.

Reply:
OK, done